docs/02-Deploying/FAQ.md

```diff
@@ -108,4 +109,10 @@ Fleet is built to run on MySQL 5.7 or above. However, particularly with AWS Auro
 
 ## What are the MySQL user requirements?
 
-The user `fleet prepare db` (via environment variable `FLEET_MYSQL_USERNAME` or command line flag `--mysql_username=<username>`) uses to interact with the database needs to be able to create, alter, and drop tables as well as the ability to create temporary tables.
\ No newline at end of file
+The user `fleet prepare db` (via environment variable `FLEET_MYSQL_USERNAME` or command line flag `--mysql_username=<username>`) uses to interact with the database needs to be able to create, alter, and drop tables as well as the ability to create temporary tables.
+
+## What is duplicate enrollment and how do I fix it?
+
+Duplicate host enrollment is when more than one host enrolls in Fleet as the same identifier (in osquery terms this is the node key). This can be caused by cloning a VM Image with an already enrolled
```

Comment:
```suggestion
Duplicate host enrollment is when more than one host enrolls in Fleet using the same identifier (hardware UUID or osquery generated UUID). This can be caused by cloning a VM Image with an already enrolled
```

The node key is used by the server to identify the host after enrollment, but it's (typically) one of those UUIDs that identifies at enrollment.